packages/cdc-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+// Checkbox.tsx
+import { ChangeEvent, ReactNode, useState } from "react";
+import "./Checkbox.scss";
+
+type CheckboxProps = {
+  id: string;
+  label?: string;
+  description?: string;
+  checkedBox?: boolean;
```

Comment:
Rename to just `checked` or `isChecked`.

Reply:
I think this should be updated too. It makes the code more readable. I like `isChecked`.